src/components/Navigator/NavigatorCard.vue

```diff
@@ -838,6 +860,9 @@ export default {
         this.focusIndex(0);
       }
     },
+    convertChildrenArrayToObject(children) {
+      return Object.fromEntries(children.map(child => [child.uid, child]));
```

Comment:
Not blocking, but as a very minor performance improvement, you may want to rewrite this in a single pass loop using `reduce` instead of looping twice (one for `fromEntries` and one for `map`